api/src/hyp3_api/openapi-spec.yml

```diff
@@ -156,9 +156,8 @@ components:
       description: The name of the Sentinel-1 granule to process.  Only SLC products with beam mode IW are supported.
 
     datetime:
-      type: integer
-      minimum: 0
-      description: a date time in unix epoch time
+      type: string
```

Comment:
would love to add a custom validator here, date-time isn't working for me

Reply:
`date-time` needs `pip install strict-rfc3339`

Reply:
I have it and it wasn't working